pipeline_annotate.sh

```diff
@@ -403,14 +409,20 @@ executeAnnotSv() {
 
   args=()
   args+=("-SVinputFile" "${inputFilePath}")
-  args+=("-outputFile" "${outputFilePath}")
+  args+=("-outputFile" "${outputFilePath}.tsv")
   args+=("-genomeBuild" "${assembly}")
   args+=("-typeOfAnnotation" "split")
-  #TODO fix phenotypes (see VIBE)
+
   if [ -n "${phenotypes}" ]; then
-    args+=("-hpo" "${phenotypes}")
-  fi
+    declare -A UNIQUE_PHENOTYPES
+    get_unique_phenotypes "${phenotypes}"
 
+    if [[ ${#UNIQUE_PHENOTYPES[@]} -gt 0 ]]
+    then
+      joined_phenotypes=$(join_arr , "${!UNIQUE_PHENOTYPES[@]}")
```

Comment:
consistency, quote all strings:
```suggestion
      joined_phenotypes=$(join_arr "," "${!UNIQUE_PHENOTYPES[@]}")
```